docs/01-Configuration.md

```diff
@@ -39,11 +39,11 @@ Configuration related interactions with the Cosmos chain in question
 
 Type: float
 
-Multiplied by estimated gas fee per transaction. It is recommended to set this to a value above 1.0 to avoid gas estimation issues.
+Multiplied by estimated gas fee per transaction. Currently Sommelier fee requirements are 0.0 so this can be left as default.
```

Comment:
I think we'd want to revert the changes to this doc.

Reply:
I wish I had written down why I did this, I feel like it was a bug I noticed. Will undo for now.

Reply:
The other elements in this doc are suggesting a gas multiplier of 1.0. The current default is 1.1 to avoid the failing message bug, and we want to make sure we don't suggest 1.0.